src/ostorlab/agent/message/proto/v3/asset/agent/agent.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto2";
+
+package v3.asset.agent;
+
+message Message {
+  required string key = 1;
+  required string version = 2;
+  required string docker_file_path = 3;
+  required string docker_build_root = 4;
+  required int32 organisation = 5;
+  required string provider = 6;
```

Comment:
All the fields below here are incorrect. You the gitrepo location and docker location. and all are optional.